### ISSUE TITLE
error in install.py

### ISSUE THREAD

**Issue by milwd** [nv-legate/legate.core#107]
greetings, I'm trying to install legate.core on jetson nano (CUDA arch-5.3 and version-10.2.3). I have also pre-installed numpy_1.19, cffi, pyarrow and it's got g++.
I tried to install from source -and with gpu and cuda support- using file : install.py
but I'm getting this error : 
sudo python3 install.py --cuda --with-cuda /usr/local/cuda-10 --arch maxwell --install-dir /usr/local/legate/

Using python lib and version: /usr/lib/python3.6/config-3.6m-aarch64-linux-gnu/libpython3.6m.so, 3.6.9
error: pathspec 'control_replication' did not match any file(s) known to git.

Traceback (most recent call last):
  File "install.py", line 946, in <module>
    driver()
  File "install.py", line 942, in driver
    install(unknown=unknown, **vars(args))
  File "install.py", line 651, in install
    update_legion(legion_src_dir, branch=legion_branch)
  File "install.py", line 216, in update_legion
    git_update(legion_src_dir, branch=branch)
  File "install.py", line 150, in git_update
    verbose_check_call(["git", "checkout", branch], cwd=repo_dir)
  File "install.py", line 85, in verbose_check_call
    subprocess.check_call(*args, **kwargs)
  File "/usr/lib/python3.6/subprocess.py", line 311, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['git', 'checkout', 'control_replication']' returned non-zero exit status 1

**Comment by lightsighter** [comment]
Would you mind trying to upgrade to Python 3.7 or later? Python 3.7 is the minimum version of Python that we support for the Legate Core at the moment. Thanks.

https://github.com/nv-legate/legate.core/blob/branch-22.01/conda/core_dev.yml#L10

**Comment by milwd** [comment]
It was a problem of Legion gitlab-repository not being cloned correctly, and after fixing it, legate installed completely and import-checked :).
Regarding Python version, I did run into a problem with it (install.py->line 31 sys.stdout.reconfigure(line_buffering=True)) which I'm aware, was implemented in Python=>3.7. I commented that line (witnessed a flushedless terminal :)) and it went On working.
*same thing for cuNumeric, which was my intent in the first step
I hope it's okai. thank u.
